src/box/lua/config/applier/credentials.lua

```diff
@@ -653,6 +653,33 @@ local function sync_privileges(credentials, obj_to_sync)
     end)
 end
 
+local schema_is_upgraded = false
+local schema_is_upgraded_cond
+
+local function update_schema_upgraded_status(version)
+    local schema_version = version or mkversion.get()
+    schema_is_upgraded = schema_version == mkversion.get_latest()
```

Comment:
Actually, no. Because `schema_needs_upgrade()` is still true, when the latest version schema is loaded and `on_replace` is fired. So I had to create a new variable. Probably this behaviour should be changed?

Reply:
I suppose, it'll be way to difficult to change. I'm ok with new variable here